financialaid/models.py

```diff
@@ -0,0 +1,110 @@
+"""
+Models for the Financial Aid App
+"""
+
+from django.contrib.auth.models import User
+from django.db import (
+    models,
+    transaction,
+)
+from jsonfield import JSONField
+
+from courses.models import (
+    CourseRun,
+    Program,
+)
+
+
+class Tiers(models.Model):
+    """
+    The possible tiers to be used
+    """
+    name = models.TextField()
+    description = models.TextField()
+
+
+class TierProgram(models.Model):
+    """
+    The tiers for discounted pricing assigned to a program
+    """
+    program = models.ForeignKey(Program, null=False)
+    tier = models.ForeignKey(Tiers, null=False)
+    value = models.IntegerField(null=False)
+    current = models.BooleanField(null=False, default=False)
+
```

Comment:
Given what we know about the algorithm, financial aid tiers are based purely on income (country only comes into play for auto-approval, and is not used in determining the tier for which a user goes into review).  Given this, I think we should add an income threshold to the TierProgram model.  Then, we can assign tiers to users by selecting the tier for a program with the lowest threshold that is still greater than that user's income